runner.rb

```diff
@@ -1,42 +1,70 @@
-require_relative 'account'
-require_relative 'user'
+require_relative 'bank'
 
-def make_account(name)
-  Account.new({ owner: name })
-end
+# SauceBank runner class
+class Runner
+  attr_accessor :customer
+  def initialize
+    @bank = Bank.new
+    @bank.create_account('latoya')
+    @magic_word = ''
+    welcome
+  end
 
-def print_menu(customer)
-  puts <<-EOT
-    #{customer.owner.name},
-    would you like to:
-    - see your balance, (balance)
-    - make a withdrawal, (withdraw) or
-    - make a deposit? (deposit)
-  EOT
-end
+  def confirm_customer(name)
+    if @bank.has_account?(name)
+      @customer = find_account(name).first
+    else
+      puts "looks like you don't have an account, No Worries, I'll make one!"
+      @customer = @bank.create_account(name)
```

Comment:
If you're creating an account, why did you choose to name this `customer`? It seems like an account, a customer, and a user, are all very similar objects.  Do you need all 3?

Reply:
I wanted to make sure there was separation, starting with a `User` and an `Account. eventually, a `User` should be able to have multiple `Accounts`, so I didn't want to have the user nestled into the `Account` Class.

I'd like `confirm_customer` to bring up a `User` that can access their multiple `Accounts` through `Bank`